packages/asset-swapper/src/utils/market_operation_utils/orders.ts

```diff
@@ -202,6 +203,8 @@ export function getErc20BridgeSourceToBridgeSource(source: ERC20BridgeSource): s
             return encodeBridgeSourceId(BridgeProtocol.AaveV2, 'AaveV2');
         case ERC20BridgeSource.Compound:
             return encodeBridgeSourceId(BridgeProtocol.Compound, 'Compound');
+        case ERC20BridgeSource.Geist:
+            return encodeBridgeSourceId(BridgeProtocol.Geist, 'Geist');
```

Comment:
If this is a 1:1 fork you can use `BridgeProtocol.AaveV2` just like we have done with the gazillion Uniswap V2 forks above 😄 
```suggestion
            return encodeBridgeSourceId(BridgeProtocol.AaveV2, 'Geist');
```